packages/vuetify/src/components/VVirtualScroll/VVirtualScroll.tsx

```diff
@@ -73,10 +75,22 @@ export const VVirtualScroll = genericComponent<new <T, Renderless extends boolea
     useToggleScope(() => props.renderless, () => {
       onMounted(() => {
         containerRef.value = getScrollParent(vm.vnode.el as HTMLElement, true)
-        containerRef.value?.addEventListener('scroll', handleScroll)
+        if (containerRef.value === document.documentElement) {
+          document.addEventListener('scroll', handleScroll, { passive: true })
+          document.addEventListener('scrollend', handleScrollend)
+        } else {
+          containerRef.value?.addEventListener('scroll', handleScroll, { passive: true })
+          containerRef.value?.addEventListener('scrollend', handleScrollend)
+        }
```

Comment:
What's the difference here? If `containerRef.value` equals `document.documentElement`, can't you just do `containerRef.value?.addEventListener` then as well?

Reply:
Stolen from #17813: https://github.com/vuetifyjs/vuetify/pull/17813/files/d871cda1ae7b935f50597822b4656c8252e82d7e#r1271301766

documentElement has `scrollHeight > clientHeight` and `scrollTop` that reflects page scroll position, but it doesn't emit a scroll event.